pkg/reconciler/pipelinerun/resources/apply.go

```diff
@@ -30,6 +30,11 @@ import (
 	"github.com/tektoncd/pipeline/pkg/substitution"
 )
 
+const (
+	resultsParseNumber              = 4
```

Comment:
Could you please add a comment describing what these are

Reply:
sure!